expression/builtin.go

```diff
@@ -781,6 +781,7 @@ var funcs = map[string]functionClass{
 	ast.VitessHash:      &vitessHashFunctionClass{baseFunctionClass{ast.VitessHash, 1, 1}},
 	ast.UUIDToBin:       &uuidToBinFunctionClass{baseFunctionClass{ast.UUIDToBin, 1, 2}},
 	ast.BinToUUID:       &binToUUIDFunctionClass{baseFunctionClass{ast.BinToUUID, 1, 2}},
+	ast.TidbShard:       &tidbShardFunctionClass{baseFunctionClass{ast.TidbShard, 1, 1}},
```

Comment:
`TiDBShard` is more proper. https://github.com/golang/go/wiki/CodeReviewComments#initialisms